src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/IProjectSnapshotExtensions.cs

```diff
@@ -37,6 +39,17 @@ public static RazorProjectInfo ToRazorProjectInfo(this IProjectSnapshot project,
             documents: documents.DrainToImmutable());
     }
 
+    public static string ToRazorProjectInfoString(this IProjectSnapshot project, string serializedFilePath)
```

Comment:
Consider renaming this to make it clear that the string that's returned is base64-encoded. Make something like `ToBase64EncodedProjectInfo`?

Reply:
Likely a temporary method until we refactor serialization, but I renamed as you suggested